src/main/java/frc/robot/commands/drivetrain/DriveToNote.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands.drivetrain;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.utils.DriverController;
+import frc.robot.utils.RotationSupplier;
+
+public class DriveToNote extends DriveToPosePID{
+    public DriveToNote(Drivetrain drivetrain, DriverController driverController, RotationSupplier rotationSupplier){
+        super(
+            drivetrain,
+            driverController,
+            () -> new Pose2d(0, 0, new Rotation2d(rotationSupplier.getValue())), 
+            new boolean[]{false, false, true}, 
```

Comment:
As, of right now, this just has the robot getting into the right rotational state to pick up the note. It does not actually translate the robot across the field.

Reply:
Easy fix would be to add the rotation to the robot's